kubectl-cloudflow/cmd/configure_command.go

```diff
@@ -23,17 +23,19 @@ func init() {
 	configureCMD.cmd = &cobra.Command{
 		Use:   "configure",
 		Short: "Configures a deployed Cloudflow application.",
-		Example: `kubectl cloudflow configure my-app mystreamlet.hostname=localhost
+		Example: `kubectl cloudflow configure my-app --conf my-config.conf
 
-TODO DOCUMENT
+Configuration files in HOCON format can be passed through with the --conf flag. All configuration files are merged in the order that they are passed through.
```

Comment:
> All configuration files are merged in the order that they are passed through.
You may want to say which order takes precedence. (first wins/last wins)

Reply:
That is the next line? 
> All configuration files are merged in the order that they are passed through.

Reply:
Or well, I can add a line for that to be exact.

Reply:
Added some more text to explain this.